### PR TITLE
feat(atomic): Added Tailwind CSS to Atomic

### PR DESCRIPTION
**Pull request by nrawji**
Added some sample styles, and themeing just to show how it would work.

### CODE REVIEW

**Review by olamothe — COMMENTED** [review]
What's the status on the problem you had for dev mode + purge css ? Is that still a problem ?

**Comment by nrawji** [comment]
> What's the status on the problem you had for dev mode + purge css ? Is that still a problem ?

I ended up using the `stencil-tailwind` library I mentioned at standup. It works well with purge but like I said, since its not really super well supported if we encounter issues in the future, we'll probably have to either submit a PR ourselves or fork of the original repo.

File: packages/atomic/stencil.config.ts [hunk 1]
```diff
@@ -36,7 +39,12 @@ export const config: Config = {
   devServer: {
     reloadStrategy: 'pageReload',
   },
-  plugins: [inlineSvg()],
+  plugins: [
+    inlineSvg(),
+    stencilTailwind({
```
Comment (samisayegh) [inline packages/atomic/stencil.config.ts:44]: I'm wondering what it is about this plugin that makes tailwind work inside the shadow dom without javascript heap-out-of-memory issues. I find the current approach really elegant, but I am worried about running into issues as we use tailwind's different features. For example, it seems like the plugin does not work with responsive classes which would be critical for us: https://github.com/jrowlingson/stencil-tailwind/issues/13

I wonder if there is another solution that has more community support. I found this write-up, could it be an alternative approach?
- https://dev.to/43081j/using-tailwind-at-run-time-with-web-components-47c
- https://github.com/tw-in-js/twind/

**Comment by nrawji** [comment]
So I found a solution that works, if we use `postcss-at-import` and move `purgecss` outside of tailwind (i.e. call it manually through postcss) we can use tailwind without the use of any Stencil specific plugins. This is probably the most flexible solution and the one with the best long term support.

The only downside is build times still aren't as quick as the plugin. I usually find that the initial build with this takes about 25 seconds and then any subsequent changes take about 15-20 seconds. Its not a super long wait but its enough to be irritating when you're trying to fix a small UI issue. But in the grand scheme of things I think this is a pretty small tradeoff.

File: packages/atomic/src/index.html [hunk 2]
```diff
@@ -8,6 +8,16 @@
     <script type="module" src="/build/atomic.esm.js"></script>
     <script nomodule src="/build/atomic.js"></script>
     <link rel="stylesheet" href="/build/atomic.css" />
+    <!-- Initialization example
```
Comment (ThibodeauJF) [inline packages/atomic/src/index.html:11]: you can remove that comment

**Review by olamothe — APPROVED** [review]

**Pull request merged**